backend/news/migrations/0001_initial.py

```diff
@@ -26,4 +27,14 @@ class Migration(migrations.Migration):
                 'abstract': False,
             },
         ),
+        migrations.CreateModel(
+            name='Article',
+            fields=[
+                ('post_title', models.CharField(max_length=255, primary_key=True, serialize=False)),
+                ('image_link', models.URLField(max_length=255)),
```

Comment:
made new migration file, after deleting previous ones because of database error:

[](https://stackoverflow.com/questions/65895320/how-do-i-fix-django-db-utils-databaseerror
)

also reinstalled a specific version of sql-parse because of database dependency. `sqlparse==0.2.4`